code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -317,3 +317,4 @@
 
 /obj/item/gun/projectile/automatic/lasercarbine/update_icon_state()
 	icon_state = "lasercarbine[magazine ? "-[CEILING(get_ammo(0)/5, 1)*5]" : ""]"
+	item_state = "lasercarbine[magazine ? "-[CEILING(get_ammo(0)/5, 1)*5]" : ""]"
```

Comment:
why the `"lasercarbine[magazine ? "-[CEILING(get_ammo(0)/5, 1)*5]" : ""]"` hasn't been turned into a define yet is beyond me (this is a comment in general, not specific to this PR)